app/views/favorites/index.html.erb

```diff
@@ -1,12 +1,15 @@
 <header><h2 class='page-title'>Favorite Pets</h2></header>
 <section class='cards-container'>
-  <% unless @favorite_pets %>
+  <% if !@favorite_pets %>
+    <%= "You haven't favorited any pets yet." %>
+  <% elsif @favorite_pets.empty? %>
     <%= "You haven't favorited any pets yet." %>
   <% else %>
     <% @favorite_pets.each do |pet| %>
       <article id="pet-<%= pet.id %>" class='pet card'>
         <p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p>
         <img src=<%= pet.image %> />
+        <p><%= button_to "Remove from favorites", "/favorites/#{pet.id}", method: :delete %>
```

Comment:
I had quite a bit of trouble with the logic here, this looks messy but it works, maybe could be refactored